src/index.html

```diff
@@ -3,12 +3,51 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css?family=Roboto:500&display=swap" rel="stylesheet" >
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <div class="wrapper">
+      <header class="header">
+        <div class="header__container">
+          <a href="#home" class="header__logo">
+            <img src="images/logo.png" alt="logo">
+          </a>
+          <nav class="header__menu menu">
+            <ul class="menu__list">
+              <li class="menu__item ">
+                <a href="#" class="is-active menu__link">Apple</a>
+              </li>
+              <li class="menu__item ">
+                <a href="#" class="menu__link">Samsung</a>
+              </li>
+              <li class="menu__item">
+                <a href="#" class="menu__link">Smartphones</a>
+              </li>
+              <li class="menu__item ">
+                <a href="#"
+                class="menu__link"
+                data-qa="hover">Laptops & Computers</a>
```

Comment:
as i undertand we can somehow make formating  automaticaly. so that we just it can be fixed just by saving page, but i can`t find how exactly i can do it